kvconfig.yml

```diff
@@ -0,0 +1,11 @@
+routes:
+  unknown-error-alerts:
+    matchers:
+      title: ["stats"]
+    output:
+      type: "alerts"
+      series: "kinesis-consumer.notifications.unknown-error"
+      dimensions: [ ]
+      # 0 for ongoing; 1 for failed; -1 for cancelled
```

Comment:
I'm confused how this can be a `counter` if it allows a value of -1. Might you link me to more information on how this works?

Reply:
derp this is a bad comment.